blockly/msg/json/fr.json

```diff
@@ -324,5 +377,120 @@
 	"PROCEDURES_CREATE_DO": "Créer '%1'",
 	"PROCEDURES_IFRETURN_TOOLTIP": "Si une valeur est vraie, alors renvoyer une seconde valeur.",
 	"PROCEDURES_IFRETURN_HELPURL": "http://c2.com/cgi/wiki?GuardClause",
-	"PROCEDURES_IFRETURN_WARNING": "Attention : Ce bloc pourrait n’être utilisé que dans une définition de fonction."
+	"PROCEDURES_IFRETURN_WARNING": "Attention : Ce bloc pourrait n’être utilisé que dans une définition de fonction.",
+	"ARD_TYPE_CHAR": "Charactère",
+	"ARD_TYPE_TEXT": "Texte",
+	"ARD_TYPE_BOOL": "Booléen",
+	"ARD_TYPE_SHORT": "Entier court",
+	"ARD_TYPE_NUMBER": "Entier",
+	"ARD_TYPE_LONG": "Entier long",
+	"ARD_TYPE_DECIMAL": "Décimal",
+	"ARD_TYPE_ARRAY": "Tableau",
+	"ARD_TYPE_NULL": "Null",
+	"ARD_TYPE_UNDEF": "Non défini",
+	"ARD_TYPE_CHILDBLOCKMISSING": "Dépendance manquante",
+	"ARD_HIGH": "HAUT",
+	"ARD_LOW": "BAS",
+	"ARD_ANALOGREAD": "Lecture du signal analogique #",
+	"ARD_ANALOGREAD_TIP": "Valeur de retour entre 0 et 1024",
+	"ARD_ANALOGWRITE": "Ecriture du signal analogique #",
+	"ARD_ANALOGWRITE_TIP": "Write analog value between 0 and 255 to a specific PWM Port",
```

Comment:
Looks like this one fell through the cracks, would it be too much trouble to add it as well?